src/styles/shared/_typography.scss

```diff
@@ -141,15 +141,8 @@ $typography-condensed: em(640px);
   }
 }
 
-@mixin text-emphasis-placeholder() {
```

Comment:
This file is part of our sass public API so this is a breaking change. Remember to document it.